### PR TITLE
Login as main page

### PR DESCRIPTION
**Pull request by rlajous**
## Summary

-Main page is login now

## Trello Card 
https://trello.com/c/Um2FXkH1/44-cambiar-el-punto-de-start-de-la-app-de-appjs-al-login

### CODE REVIEW

File: generators/app/templates/src/app/components/Routes/components/AuthenticatedRoute.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ import PropTypes from 'prop-types';
 
 import Routes from '../../../../constants/routes';
 
-const DEFAULT_PUBLIC_ROUTE = Routes.LOGIN;
-const DEFAULT_PRIVATE_ROUTE = Routes.HOME;
+const DEFAULT_PUBLIC_ROUTE = Routes.HOME;
```
Comment (HarryPerez) [inline generators/app/templates/src/app/components/Routes/components/AuthenticatedRoute.js]: Is this ok? Home can not be the default public route. You need to be logged in order to access private routes. Check this.

**Review by HarryPerez — CHANGES_REQUESTED** [review]

**Review by pabloccid — COMMENTED** [review]
I think you inverted the logic. It was ok as it was. The login screen was already the default public route

File: generators/app/templates/src/app/components/Routes/components/AuthenticatedRoute.js [hunk 2]
```diff
@@ -72,15 +72,12 @@ AuthenticatedRoute.defaultProps = {
...
 };
 
 AuthenticatedRoute.propTypes = {
   ...Route.propTypes, // eslint-disable-line react/forbid-foreign-prop-types
-  /*
-   * TODO Add this if you need it
-   * currentUser: PropTypes.bool,
-   */
+  currentUser: PropTypes.bool,
   initialized: PropTypes.bool,
```
Comment (pabloccid) [inline generators/app/templates/src/app/components/Routes/components/AuthenticatedRoute.js]: dilit dis

File: generators/app/templates/src/app/components/Routes/components/AuthenticatedRoute.js [hunk 3]
```diff
@@ -72,15 +72,12 @@ AuthenticatedRoute.defaultProps = {
    * currentUser: false,
```
Comment (pabloccid) [inline generators/app/templates/src/app/components/Routes/components/AuthenticatedRoute.js]: dilit dis

**Review by pabloccid — APPROVED** [review]

**Review by HarryPerez — APPROVED** [review]

**Pull request merged**